### DISCUSSION TITLE
Embed namespace in the class definition

### DISCUSSION THREAD

**Discussion by foximoxi** [dotnet/csharplang#8021] [Language Ideas]
I would like to propose additional class definition with embedded namespace
Instead of:

```
namespace MyNamespace;

public class Foo
{
    //...code
}
```

```
public class MyNamespace.Foo
{
     //...code
}
```

There is as lot of situations where it is required to produce simple classes (e.g. models) and this would save one or two lines which improves readability of the code

**Comment by ufcpp** [comment]
https://github.com/dotnet/csharplang/discussions/711

**Comment by AndersKehlet** [comment]
I think it made sense to consider prior to file-scoped namespaces, but at present you'd literally just be saving a single line. Honestly doesn't seem worthwhile to me.
